src/net_processing.cpp

```diff
@@ -124,6 +124,9 @@ namespace {
     /** Number of peers from which we're downloading blocks. */
     int nPeersWithValidatedDownloads = 0;
 
+    /** Number of outbound peers with fProtectFromDisconnect. */
+    int nOutboundPeersWithProtectFromDisconnect = 0;
```

Comment:
Per developer-notes, we should follow the `m_snake_case` naming convention for all member variables. This also applied to variables below.